[Keyboard] add MT64RGB (#10865)
qmk/qmk_firmware: keyboards/mt64rgb/readme.md

# mt64rgb 

![mt64rgb](https://i.imgur.com/kxGGUg9.jpg?1)

A 60% keyboard PCB made by MT.
It supports an ANSI-based 64-key layout, with hotswap sockets and per-key RGB backlight.

> * Keyboard Maintainer: [MT](https://github.com/704340378)

Reviewer: ```suggestion

* Keyboard Maintainer: [MT](https://github.com/704340378)
```

# mt64rgb 

![mt64rgb](https://i.imgur.com/kxGGUg9.jpg?1)

A 60% keyboard PCB made by MT.
It supports standard 64 keyboard，with hotswap and RGB backlight.

>  Keyboard Maintainer: [MT](https://github.com/704340378)

Reviewer: ```suggestion
* Keyboard Maintainer: [MT](https://github.com/704340378)

```
Please also change these files to Unix line endings.

# mt64rgb 

![mt64rgb](https://i.imgur.com/kxGGUg9.jpg?1)

A 60% keyboard PCB made by MT.

> It supports standard 64 keyboard，with hotswap and RGB backlight.

Reviewer 2: ```suggestion
It supports an ANSI-based 64-key layout, with hotswap sockets and per-key RGB backlight.
```

Author: will do ,thank you
